README.md

```diff
@@ -549,6 +549,111 @@ cd test/manual/k8s_crds
 ```
 This test currently does not run as part of the test suite.
 
+## Profiling
+Profiling can be used to monitor the impact of Secretless on CPU and Memory consumption. Currently, Secretless supports two types- CPU and Memory.
+
+**Prerequisites:**
+- **Homebrew**
+- Graphviz to visualize profiling results: ```$ brew install graphviz```
```

Comment:
I guess instructions to install `psql` might be necessary too since they'll need it on their local machine.